pkg/nsx/services/subnetport/builder.go

```diff
@@ -65,6 +65,18 @@ func (service *SubnetPortService) buildSubnetPort(obj interface{}, nsxSubnet *mo
 	}
 	namespace_uid := namespace.UID
 	tags := util.BuildBasicTags(getCluster(service), obj, namespace_uid)
+
+	var tagsFiltered []model.Tag
+	for _, tag := range tags {
+		if isVmSubnetPort && *tag.Scope == common.TagScopeNamespaceUID {
```

Comment:
for image-fetcher subnetport, it does not filter the vm_namespace tag.

Reply:
fixed